### PR TITLE
Update version and change log for 0.15.0

### PR DESCRIPTION
**Pull request by prusse-martin**

### CODE REVIEW

File: src/alfasim_sdk/_internal/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.14.0"
```
Comment (prusse-martin) [inline src/alfasim_sdk/_internal/version.py:1]: This file should usualy be already with the correct version.

**Comment by prusse-martin** [comment]
`ALFAsim SDK / test (3.6, ubuntu-latest) (pull_request)` is failing because python 3.6 is missing on ubuntu 2022 (latest).

**Comment by prusse-martin** [comment]
The flake 8 error will be fixed in #279 .

File: src/alfasim_sdk/_internal/version.py [hunk 2]
```diff
@@ -1 +1 @@
...
+__version__ = "0.15.0"
```
Comment (prusse-martin) [inline src/alfasim_sdk/_internal/version.py:1]: This file should usually be already with the correct version

**Review by BeneBr — APPROVED** [review]

**Review by dettmerramon — APPROVED** [review]

**Comment by prusse-martin** [comment]
> `ALFAsim SDK / test (3.6, ubuntu-latest) (pull_request)` is failing because python 3.6 is missing on ubuntu 2022 (latest).

I did cherry-pick the commit from master fixing this.

**Pull request merged**